crates/nu-command/src/filters/append.rs

```diff
@@ -16,12 +16,16 @@ impl Command for Append {
 
     fn signature(&self) -> nu_protocol::Signature {
         Signature::build("append")
-            .required("row", SyntaxShape::Any, "the row to append")
+            .required("row", SyntaxShape::Any, "the row (or list/table) to append")
```

Comment:
I considered renaming the `row` parameters but couldn't think of a good name. Can do so if anyone has a good idea.